### PR TITLE
FTBFS fix for loongarch batch #6

### PR DESCRIPTION
**Pull request by jiegec**
Topic Description
-----------------

- messagelib: add loongarh64 to FAIL_ARCH
- nautilus-python: add gtk-doc to BUILDDEP
- mozjpeg: mark noarch to fix FTBFS
- python-lz4: disable building for python2 due to FTBFS
- lz4: fix meson usage

Package(s) Affected
-------------------

- lz4: 1:1.9.4-1
- messagelib: 22.08.3
- mozjpeg: 1:3.1-1
- nautilus-python: 1.2.3-3
- python-lz4: 2.1.6-5

Security Update?
----------------

No

Build Order
-----------

```
#buildit lz4 python-lz4 mozjpeg nautilus-python messagelib
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

**Experimental Architectures**

- [x] MIPS R6 64-bit (Little Endian) `mips64r6el`

### CODE REVIEW

**Comment by jiegec** [comment]
python-lz4 FTBFS on python2:

```
[INFO]:  Building Python (PyPI) package using /usr/bin/python2 ...
WARNING: The pip package is not available, falling back to EasyInstall for handling setup_requires/test_requires; this is deprecat
ed and will be removed in a future version.
  File "build/bdist.linux-loongarch64/egg/pkgconfig/pkgconfig.py", line 281
    flags = _query(packages, *os_opts, *_build_options(option, static=static))
                                       ^
SyntaxError: invalid syntax

zip_safe flag not set; analyzing archive contents...

Installed /var/cache/acbs/build/acbs.yrxofvl3/lz4-2.1.6/.eggs/pkgconfig-1.5.5-py2.7.egg
WARNING: The pip package is not available, falling back to EasyInstall for handling setup_requires/test_requires; this is deprecat
ed and will be removed in a future version.
Searching for setuptools_scm
Reading https://pypi.org/simple/setuptools_scm/
Downloading https://files.pythonhosted.org/packages/eb/b1/0248705f10f6de5eefe7ff93e399f7192257b23df4d431d2f5680bb2778f/setuptools-
scm-8.0.4.tar.gz#sha256=b5f43ff6800669595193fd09891564ee9d1d7dcb196cab4b2506d53a2e1c95c7
Best match: setuptools-scm 8.0.4
Processing setuptools-scm-8.0.4.tar.gz
Traceback (most recent call last):
  File "/var/cache/acbs/build/acbs.yrxofvl3/lz4-2.1.6/setup.py", line 200, in <module>
    'Programming Language :: Python :: 3.7',
  File "/usr/lib/python2.7/site-packages/setuptools/__init__.py", line 144, in setup
    _install_setup_requires(attrs)
  File "/usr/lib/python2.7/site-packages/setuptools/__init__.py", line 139, in _install_setup_requires
    dist.fetch_build_eggs(dist.setup_requires)
  File "/usr/lib/python2.7/site-packages/setuptools/dist.py", line 721, in fetch_build_eggs
    replace_conflicting=True,
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 782, in resolve
```

The current pr disables building for python2, but maybe it can be fixed.

File: app-utils/lz4/spec [hunk 1]
```diff
@@ -1,4 +1,6 @@
 VER=1.9.4
+REL=1
 SRCS="tbl::https://github.com/lz4/lz4/archive/v$VER/lz4-$VER.tar.gz"
```
Comment (MingcongBai) [inline app-utils/lz4/spec]: Use Git tag here.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: app-utils/lz4/spec (hunk 1 above)
Comment (jiegec) [reply app-utils/lz4/spec]: Fixed

**Review by MingcongBai — APPROVED** [review]

**Comment by jiegec** [comment]
lz4, python-lz4 worked on amd64.

**Pull request merged**